architecture/core_objects/routing.adoc

```diff
@@ -119,10 +120,16 @@ file.
 
 ====
 
-Unsecure routes are likely faster to set up, as they are the default configuration, but secure routes offer greater security for information to remain private.
+Unsecure routes are can be faster to set up, as they are the default
```

Comment:
@tnguyen-rh Ok. I asked around a little, and it seems "unsecured" is indeed the better option. But does that mean all the instances of "secure" need to be changed to "secured"? I think my thinking was that "secure and unsecure routes" were a _type_, or route, so needed caps. Something about being Proper nouns in a way.